### PR TITLE
update about-logging.md

### PR DESCRIPTION
**Pull request by shreyas1599**
Made the changes suggested in #46, #48, #49, and #50.

### CODE REVIEW

File: logging/about-logging.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ The description section gives the reader all the key information they need to un
...
 
-You don't need to describe the mandatory logging parameters sent by default to the log pipeline. These parameters should be defined in higher-level reference documentation for your project.
+You don't need to describe the mandatory logging parameters sent by default to the log pipeline. Define these paramaters in the higher-level reference documentation of your project.
 
 Only describe the parameters unique to your application or common parameters that are being reused from other applications.
 
 ### About the "Logging example" Section
 
-If you get called late at night and you're trying to debug a problem, not knowing how to interpret a logfile because there is no example of what to expect is extremely frustrating.
+It can be extremely frustrating to get a call late at night when you're trying to debug a problem, if you don't provide an example of how to interpret a logfile.
```
Comment (jaredmorgs) [inline logging/about-logging.md:23]: My initial wording of this was awkward and this hasn't helped you understand what I was suggesting in this sentence.

Maybe something like:

"Not having good log file examples can make it frustrating to debug a problem when you are on-call."

File: logging/about-logging.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ The description section gives the reader all the key information they need to un
...
 
 If you can describe what you will see in the log file, it will be immensely helpful to the person trying to fix a critical issue at 2 am in the morning.
 
 ### About the "Log properties" Section
 
-If you have the concept of mandatory and app-specific log properties, you can specify app-specific log properties in this section.
+You can specify app-specific log properties in this section.
```
Comment (jaredmorgs) [inline logging/about-logging.md:29]: Much simpler. Thanks.

File: logging/about-logging.md (hunk 1 above)
Comment (shreyas1599) [reply logging/about-logging.md]: @jaredmorgs I made the change. The next sentence too sounds a bit odd. I changed that as well.

File: logging/about-logging.md [hunk 3]
```diff
@@ -14,19 +14,19 @@ The description section gives the reader all the key information they need to un
...
 
 Only describe the parameters unique to your application or common parameters that are being reused from other applications.
 
 ### About the "Logging example" Section
 
-If you get called late at night and you're trying to debug a problem, not knowing how to interpret a logfile because there is no example of what to expect is extremely frustrating.
+Not having good log file examples can make it hard to debug a problem when you are on-call.
 
-If you can describe what you will see in the log file, it will be immensely helpful to the person trying to fix a critical issue at 2 am in the morning.
+Proper description of the content in a log file can make it easier for the person on-call to solve the problem.
```
Comment (jaredmorgs) [inline logging/about-logging.md:25]: I like the rewording here as well.

File: logging/about-logging.md (hunk 1 above)
Comment (jaredmorgs) [reply logging/about-logging.md]: Yep that did need some tweaking as well. Thanks for the fix.

**Pull request merged**